tests/unit/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * checkboxradio_core.js
+ */
+
+
+(function($) {
+
+module("Checkboxradio: core");
+test("Checkbox", function() {
+	expect( 4 );
+	var input = $("#check"),
+		label = $("label[for=check]");
+	ok( input.is( ":visible" ) );
+	ok( !label.hasClass(".ui-button)") );
+	input.checkboxradio();
+	equal( input.is( ".ui-helper-hidden-accessible.ui-checkboxradio" ), true,
+		"Input has proper classes" );
+	equal( label.is( ".ui-icon-beginning.ui-button.ui-widget.ui-checkboxradio-label.ui-corner-all" ),
+		true, "Label has proper classes" );
+});
+
+test("Radios", function() {
+	expect( 4 );
+	var inputs = $("#radio0 input"),
+		labels = $("#radio0 label");
+	ok( inputs.is(":visible") );
+	ok( labels.is(":not(.ui-button)") );
+	inputs.checkboxradio();
+	ok( inputs.is(".ui-helper-hidden-accessible") );
+	ok( labels.is(".ui-button") );
+});
+
+function assert(noForm, form1, form2) {
```

Comment:
This should also verify that no other element in each group has the active class.